lib/executor.js

```diff
@@ -175,6 +175,7 @@ function setupDataSources(app, instructions) {
       useEnvVars: true,
     };
     obj = getUpdatedConfigObject(app, obj, opts);
+    obj.lazyConnect = true;
```

Comment:
I think `boot` should trigger `connect` by default. We can set an option to disable it so that swagger generation doesn't depend on live database connections.